v0/destinations/wootric/data/createResponseConfig.json

```diff
@@ -0,0 +1,44 @@
+[
+  {
+    "destKey": "score",
+    "sourceKeys": "properties.feedbackScore",
+    "sourceFromGenericMap": false,
+    "required": true
+  },
+  {
+    "destKey": "ip_address",
+    "sourceKeys": ["context.ip", "request_ip"],
+    "sourceFromGenericMap": false,
+    "required": true
+  },
+  {
+    "destKey": "origin_url",
+    "sourceKeys": "pageUrl",
+    "sourceFromGenericMap": true,
+    "required": true
+  },
+  {
+    "destKey": "text",
+    "sourceKeys": "properties.feedbackText",
+    "sourceFromGenericMap": false,
```

Comment:
where we are not mapping from genericMap, we do not need to mention `"sourceFromGenericMap": false`